### ISSUE TITLE
Error when pressing Ctrl + L in Windows 10

### ISSUE THREAD

**Issue by alelnro1** [PowerShell/PSReadLine#709]
```
-----------------------------------------------------------------------
Last 200 Keys:
 Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Bac
 c l s Enter
 UpArrow UpArrow Enter
 UpArrow Enter
 p h p Spacebar a r t i s a n Spacebar c o n f i g : c l e a r Enter
 UpArrow Enter
 UpArrow Enter
 UpArrow Enter
 UpArrow Enter
 p h p Spacebar c o m p o s e r . p h a r Spacebar d u Backspace Backspace d u m p a u t o l o a d Enter
 Enter
 Enter
 Enter
 UpArrow Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace u p d a t e Enter
 Enter
 Enter
 Enter
 Enter
 Enter
 Enter
 Enter
 UpArrow Enter
 UpArrow UpArrow UpArrow UpArrow Enter
 p h p Spacebar r t i s a n Spacebar c a Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace Backspace a r t i s a n Spacebar c a c h e : c o t i
u i o n Backspace Backspace Backspace Backspace i o n - b y m a Ctrl+l Ctrl+c c Ctrl+c c Enter
 Ctrl+c Ctrl+c Ctrl+l

Exception:
System.IO.IOException: The parameter is incorrect.

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.Console.SetWindowPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

**Comment by daxian-dbw** [comment]
Close as a duplicate of #1011

**Comment by daxian-dbw** [comment]
This issue had already been fixed by #790. The fix was included in 2.0.0-beta4 and newer versions. For more information, take a look at https://github.com/PowerShell/PSReadLine/issues/922#issuecomment-570475042.

Please try out the latest 2.0.0-rc1 version from PowerShell Gallery.
